RELEASE.md

```diff
@@ -0,0 +1,110 @@
+# Release
+
+This page describes the release process.
+
+## Prepare for release
+
+### (HouseRules only) Generate JSON for built-in rulesets
+
+Uncomment the relevant line from `ExampleRulesetExporter.cs`. Build and run the
+mod so it generates JSON files for built-in rulesets during game startup.
+
+Open and merge a PR with these JSON files added to `docs/rulesets` for
+documentation and reference purposes.
+
+## Cutting a release
+
+### Tag the release
+
+Every release is tagged with `v<major>.<minor>.<patch>-<mod-name>`,
+e.g. `v1.0.0-houserules`. Note the `v` prefix.
+
+The commit, on the `main` branch, from which to cut a release must have this
+tag. The following example assumes the semantic version "1.0.0" and the mod
+"HouseRules".
+
+```shell
+git tag -s "v1.0.0-houserules" -m "HouseRules v1.0.0"
```

Comment:
When multiple releases happen simultaneously (HouseRules & Roomfinder for example) I'm guessing that the process is to apply both tags onto the same commit. Guessing is probably not the correct thing to be doing when following documentation, so I'm asking for some clarification to be added here.

Reply:
Added clarification👍🏾.